Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+anytree = ">=2.8.0"
+stringcase = ">=1.2.0"
+deprecation = ">=2.1.0"
+pytest = ">=2.7.2"
+PyYAML = ">=5.1"
+vss-tools = {editable = true, path = "."}
+
+[dev-packages]
+
+[requires]
+python_version = "3.7"
```

Comment:
This makes it fail on Ubuntu LTS, where default Python is 3.8.5.   as ">=" does not seem to be supported maybe just use "3" ?

Reply:
We should agree on a version (for the Pipfile) so everything matches (see the `Pipfile.lock`, where exact versions are used).

While I'm totally fine to move to 3.8.5 or something else, the way to use `pipenv` is to pair it with [`pyenv`](https://github.com/pyenv/pyenv), then when you `pipenv` it will use the version.

Let's agree on a version and I'll document the `pyenv` in the `README.md`

Reply:
just did this, fixed to 3.8.5 and explained how to use `pyenv`. This helps everythone (being on Ubuntu or not) to have the same setup.

Of course, we should still test other systems in the CI, usually via a python matrix (ie: latest 3.7, 3.8, 3.9...)